.gitignore

```diff
@@ -1,4 +1,6 @@
 node_modules
+corvid_docs
```

Comment:
what is corvid_docs ?

Reply:
temporary folder
I use `tmp` module instead